core/pkg/service/flag-evaluation/connect_service.go

```diff
@@ -206,8 +219,10 @@ func (s *ConnectService) startMetricsServer(svcConf service.Configuration) error
 	s.metricsServer.Handler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		switch r.URL.Path {
```

Comment:
In my new implementation, I've used `mux` instead of this kind of switch statement.